cardano-cli/src/Cardano/CLI/Orphans.hs

```diff
@@ -32,3 +34,8 @@ instance ToJSON HashableScriptData where
       [ "hash" .= hashScriptDataBytes hsd
       , "json" .= scriptDataToJsonDetailedSchema hsd
       ]
+
+-- TODO move LocalNodeConnectInfo to cardano-api
+deriving instance Generic LocalNodeConnectInfo
```

Comment:
This can be removed, its not being used.

Reply:
removed